entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+export PYTHONPATH=".:$PYTHONPATH"
+
+mkdir -p static
+
+# For deployment, we migrate, collectstatic etc.
+if [ -z "$DEBUG" ]; then
+  python manage.py migrate
+  # python manage.py collectstatic --no-input
+  # python manage.py compilemessages
+else
+  python manage.py migrate
+  # python manage.py collectstatic --no-input
+  # python manage.py compilemessages
+  python manage.py loaddata events/fixtures/events.json
+fi
```

Comment:
I wouldn't recommend doing this. Migrations are not to auto apply and it breaks multiple workflows.

Reply:
True!! it was just to make starting the docker container a simple matter of `docker compose up`.. but no need to do something that needs undoing.